### PR TITLE
Add Mature.nl

### PR DESCRIPTION
**Pull request by Emilo2**
Includes scene and gallery scraping.

### CODE REVIEW

File: scrapers/MatureNL.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    url:
+      - mature.nl/
+    scraper: sceneScraper
+galleryByURL: *byUrl
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: &title //div[@class="box"]/h1/text()
+      Details: &details //meta[@name="description"]/@content
```
Comment (bnkai) [inline scrapers/MatureNL.yml:13]: It seems that some scenes (only found one atm) have an actual description that should be prefered over the meta
sample `https://www.mature.nl/en/update/14026/spying-on-my-older-and-taller-stepsister`
Based on the above the following selector seems to work ok. I am not sure if the `synopsis` part is there in other scenes though. We would have to find some more with actual descriptions
```yaml
      Details: &details //div/span[text()="Synopsis:"]/following-sibling::text()|//meta[@name="description"]/@content
```

File: scrapers/MatureNL.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        Name: 
+          selector: //div[@class="name"]/span[@class="col-accent"]/text()
+          postProcess:
+          - replace: 
+            - regex: ( \(EU\))$
+              with: 
+      Image:
+        selector: //span[@id="spnPageUpdateTrailer"]/a/img/@data-src
+      Date: &date
+        selector: //div[@class="box-cnt"]/div[@class="mar-t"]/text()
```
Comment (bnkai) [inline scrapers/MatureNL.yml:26]: the date selector matches 2 divs (the second being the view/watch link) and the text a few more commas (you can have a look at the console with log set to debug when scraping)
The below selector seems to get us only the needed text
```yaml
        selector:  //div[@class="box-cnt"]/div[@class="mar-t"][not (contains(a, "the full"))]/text()[1]
```

File: scrapers/MatureNL.yml (hunk 1 above)
Comment (Emilo2) [reply scrapers/MatureNL.yml:13]: Yeah found scenes with the Synopsis part with [Google](https://www.google.com/search?q=site%3Amature.nl+%22synopsis%3A%22) quite easily. 

Your suggestion seems to be good and works with both kinds of pages. I'll add the change.

File: scrapers/MatureNL.yml (hunk 2 above)
Comment (Emilo2) [reply scrapers/MatureNL.yml]: You suggestion works great

**Pull request merged**